backend/bottari/src/main/java/com/bottari/repository/BottariTemplateRepository.java

```diff
@@ -8,6 +8,15 @@
 
 public interface BottariTemplateRepository extends JpaRepository<BottariTemplate, Long> {
 
+    @Query("""
+            SELECT bt
+            FROM BottariTemplate bt
+            JOIN FETCH bt.member m
+            WHERE m.id = :id
+            ORDER BY bt.createdAt DESC
+            """)
+    List<BottariTemplate> findAllMyBottariTemplatesByMemberIdWithMember(final Long id);
```

Comment:
```suggestion
    List<BottariTemplate> findAllByMemberIdWithMember(final Long memberId);
```

혹시 이건 어떨까요?

- memberId를 통해 찾는 것이므로, `My` 나의 보따리 템플릿만을 찾는 것이 아니라고 생각
- 이미 BottariTemplateRepository 내에 있으므로, BottariTemplate을 가져온다는 것을 나타내주지 않아도 된다고 생각
- 파라미터명이 id라면, bottariTemplate의 id라고 착각하기 쉬움